src/pages/project/tests/projectContributorHelpers.test.ts

```diff
@@ -0,0 +1,210 @@
+import { describe, expect, it } from 'vitest';
+import { addContributor, AddContributorErrors } from '../helpers/projectContributorHelpers.js';
+import {
+  abcOrgAsAffiliation,
+  contributorsArrayWithProjectManager,
+  selectedPersonIdentity,
+  selectedPersonWithAffiliation,
+  selectedPersonWithoutAffiliation,
+} from './mockObjects';
+
+describe('addContributor', () => {
+  describe('when adding a project manager with affiliation', () => {
+    it.skip('if there is already a project manager in the contributors array it returns an error', async () => {
+      expect(
+        addContributor(selectedPersonWithAffiliation, contributorsArrayWithProjectManager, 'ProjectManager')
+      ).toEqual({
+        error: AddContributorErrors.ALREADY_HAS_A_PROJECT_MANAGER,
+      });
+    });
+    describe('when there exists a project contributor with the same id', () => {
+      it('it adds a new "ProjectManager" role with the added affiliation to the existing user when the existing contributor has an affiliation with the same id. The contributor keeps its role.', async () => {
+        /* */
```

Comment:
dette er ting du planlegger å skrive tester for i senere PR?

Reply:
Ja jeg laget en komplisert logikk-tabell for alle valg vi kunne gjøre i UI, og laget testcaser for alle. Noen kan oppleves som unødvendige siden valgene ikke skal påvirke hverandre, men det er jo ofte i slike edge caser vi oppdager rar oppførsel. Implementerer resten av testene i neste PR.